### ISSUE TITLE
Skip Link 

### ISSUE THREAD

**Issue by bhoppers2008** [jpmorganchase/salt-ds#2385]
### Skip Link Component Specification

**1. Component Overview**

The Skip Link component provides a way for users to quickly navigate to the main content of a webpage, bypassing repetitive navigation links. This is particularly useful for users who rely on keyboard navigation or screen readers.

**2. Requirements**

- Accessibility: The component must be fully accessible, adhering to WCAG 2.2 guidelines.
- Visibility: The skip link should be visible when focused and hidden otherwise.
- Keyboard Navigation: The skip link should be the first focusable element on the page.
- Customizability: The text and target of the skip link should be customizable.

**3. Design Specifications**

- Text: Default text should be "Skip to main content".
- Position: The link should be positioned at the top of the page, but off-screen by default.
- Focus State: When focused, the link should become visible and positioned at the top of the viewport.
- Styling: The link should have a clear visual indication when focused (e.g., a border or background color change).

**Comment by mark-tate** [comment]
Galao Goal: kickoff, complete by EOS, if we keep it small and we have capacity

**Comment by mark-tate** [comment]
@ivan-calderon  moving this to next sprint, move back to this sprint if you are working on it and don't start unless we can complete EOS

**Comment by mark-tate** [comment]
Latte Goal: kickoff, complete by EOS, if we keep it small and we have capacity

**Comment by ivan-calderon** [comment]
[Skip link site page](https://www.figma.com/design/n97IPaHxFP9d3CMvOwSkJx/Skip-link-pattern?node-id=2004-729&t=ZELSNE58qKibPsQl-1) shared for review. Kickoff to be scheduled depending of capacity.

**Comment by jake-costa** [comment]
A11y review complete, some changes are needed, comments have been added to figma [Skip link pattern site page](https://www.figma.com/design/n97IPaHxFP9d3CMvOwSkJx/Skip-link-pattern?node-id=2004-729&t=ZELSNE58qKibPsQl-1).

**Comment by ivan-calderon** [comment]
Updates under review. Figma file is including: a [draft component](https://www.figma.com/design/n97IPaHxFP9d3CMvOwSkJx/Skip-link-pattern?node-id=1-98), [characteristics](https://www.figma.com/design/n97IPaHxFP9d3CMvOwSkJx/Skip-link-pattern?node-id=2058-1638), [accessibility](https://www.figma.com/design/n97IPaHxFP9d3CMvOwSkJx/Skip-link-pattern?node-id=2062-2456&t=cWMVcZT12GcFkejK-1) info. Thanks @dplsek and @jake-costa for the feedback!

**Comment by mark-tate** [comment]
Macc Goal: review and get to a ready for dev

**Comment by origami-z** [comment]
Mocha: re-reviewer

**Comment by mark-tate** [comment]
@bhoppers2008 

Mocha Goal: review and publish by EOY

**Comment by bhoppers2008** [comment]
Page has been reviewed and is ready to be published. Action to review Fernanda's PR and assess number of changes. @ivan-calderon

**Comment by ivan-calderon** [comment]
@joshwooding could you please confirm if skip link is going to be as a component in code due to the benefits of it? I remember you recommended to have this as a component. But we can provide a pattern page documentation too. TY!

**Comment by mark-tate** [comment]
Red Eye: can we pick this up in red Eye @Fercas123 ?

We believe it will be a component, we don't have a good way to do Popper components
A re-usable component that can be used for use-cases like skip link date picker (we created an overlay component) and ultimately mega-menu

**Comment by mark-tate** [comment]
aim to push to Labs by EOS...

**Comment by navkaur76** [comment]
11/12 - KO meeting. Skip link pattern to be published, but component set needs to be created in figma. Nav to oversee component branches in current + Next, Ivan to oversee component branch in the new Figma library.

**Comment by bhoppers2008** [comment]
Component docs to be created and with examples and merged. 

@navkaur76 to create new ticket to create skip link component in new Salt DS Figma (@yunjungyeh )

**Comment by dplsek** [comment]
@ivan-calderon @yunjungyeh to create component in new library by end of sprint

**Comment by ivan-calderon** [comment]
Skip Link component added to Salt DS Figma library and shared for review. See [Skip Link branch](https://www.figma.com/design/7Fh5JbUUM84pYwLt9tpJh2/branch/QKu7NvlimkEBjDWnLXEhsH/Salt-DS-Components-and-Patterns?m=auto&node-id=52436-3772&t=as6yj8ilhq7xXkj9-1).

**Comment by bhoppers2008** [comment]
Goal: On track to publish to site with examples.

**Comment by mark-tate** [comment]
Tecno goal: Raise a  PR to merge to Core by w/e 25th Jan
Reviewed already
